frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -1,23 +1,533 @@
+          total and divide by 12 months to list a monthly average.)
+        </small>
+        {data.person.grossMonthlyIncome && (
+          <>
+            <TextInput label="Work" dataKey="person.income.work" />
+            <small className="web-form-input">
+              (Including self employment, wages, salaries, commisions, bonuses,
+              tips and overtime)
+            </small>
+            <TextInput
+              label="Rental income"
+              dataKey="person.income.rentalIncome"
+            />
+            <TextInput
+              label="Business income"
+              dataKey="person.income.businessIncome"
+            />
+            <TextInput label="Interest" dataKey="person.income.interest" />
+            <TextInput label="Dividends" dataKey="person.income.dividends" />
+            <TextInput
+              label="Retirement income"
+              dataKey="person.income.retirementIncome"
+            />
+            <small className="web-form-input">
+              (Including pensions, 401(k), IRA, etc.)
+            </small>
+            <TextInput
+              label="Workers compensation"
+              dataKey="person.income.workersCompensation"
+            />
+            <TextInput
+              label="Private disability insurance"
+              dataKey="person.income.privateDisabilityInsurance"
+            />
+            <TextInput
+              label="Social Security Disability Income"
+              dataKey="person.income.ssdi"
+            />
+            <small className="web-form-input">(SSDI)</small>
+            <TextInput
+              label="Supplemental Security Income"
+              dataKey="person.income.ssi"
+            />
+            <small className="web-form-input">(SSI)</small>
+            <TextInput
+              label="Social Security"
+              dataKey="person.income.socialSecurityOther"
+            />
+            <small className="web-form-input">(Other than SSDI or SSI)</small>
+            <TextInput
+              label="Unemployment benifits"
+              dataKey="person.income.unemploymentBenifits"
+            />
+            <TextInput
+              label="Educational benifits"
+              dataKey="person.income.educationalBenifits"
+            />
+            <small className="web-form-input">
+              (Including grants, loans, cash scholarships, etc.)
+            </small>
+            <TextInput
+              label="Veteran's benefits"
+              dataKey="person.income.veteransBenefits"
+            />
+            <TextInput label="Alimony" dataKey="person.income.alimony" />
+            <TextInput
+              label="Child Support"
+              dataKey="person.income.childSupport"
+            />
+            <TextInput
+              label="Payments from civil litigation "
+              dataKey="person.income.cilvilPayments"
+            />
+            <TextInput
+              label="Victim resitutuion"
```

Comment:
i believe there is a typo here on `restitution`

Reply:
There were a ton of typos. I fixed fixed them all.